pom.xml

```diff
@@ -9,11 +9,11 @@
 
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>redhat-dependency-analytics</artifactId>
-    <version>0.7.2-SNAPSHOT</version>
+    <version>${revision}.${changelist}</version>
     <packaging>hpi</packaging>
     <properties>
-        <revision>0.1.0</revision>
-        <changelist>-SNAPSHOT</changelist>
+        <revision>1.0.0</revision>
+        <changelist>999999-SNAPSHOT</changelist>
         <java.level>11</java.level>
```

Comment:
This is obsolete and can be removed. The plugin pom does no longer consider setting a per-plugin Java level.